roles/libvirt_manager/tasks/ocp_layout.yml

```diff
@@ -61,26 +61,80 @@
         state: directory
       loop:
         - "{{ cifmw_libvirt_manager_basedir }}/images"
+        - "{{ cifmw_libvirt_manager_basedir }}/workload"
         - "{{ cifmw_libvirt_manager_ocp_pool_dir }}"
 
     - name: Create pool in libvirt
       vars:
         action: 'create'
       ansible.builtin.import_tasks: "storage_pool.yml"
 
+    - name: Ldenny debug reate and run VMs
+      when:
+        - (
+            vm_data.value.target is defined and
+            vm_data.value.target == inventory_hostname
+          ) or
+          vm_data.value.target is undefined
+        - (vm_data.value.amount is defined and
+          (vm_data.value.amount | int) > 0) or
+          vm_data.value.amount is undefined
+      vars:
+        _matcher: >-
+          {{
+            '(cifmw-)?' ~
+            '([a-z0-9]+)-[0-9]+'
+          }}
+        _ocp_master_matcher: >-
+          {{
+            '^ocp-(\d+)$'
+          }}
+        vm_type: "{{ vm | regex_replace(_matcher, '\\2') }}"
+        vm_data: >-
+          {{
+            _ocp_layout.vms[vm_type]
+          }}
+        # If $vm is an OCP master node, capture it's index
+        # and craft the _vm_img varible, examples:
+        # ocp-0 -> ocp_master-0.qcow (match)
+        # ocp_worker-0 -> ocp_worker-0.qcow (pass)
+        # compute-1 -> compute-1.qcow (pass)
+        _vm_img: "{{ vm | regex_search(_ocp_master_matcher) | ternary(vm | regex_replace(_ocp_master_matcher, 'ocp_master-\\1.qcow2'), vm ~ '.qcow2') }}"
```

Comment:
hm... imho it should try to match the disk_file_name instead, and build the expected value. WDYT?